jenkins/packer/packer-build.jenkinsfile

```diff
@@ -0,0 +1,80 @@
+pipeline {
+    options {
+        timeout(time: 4, unit: 'HOURS')
+    }
+    agent {
+        docker {
+            label 'Jenkins-Agent-Ubuntu2004-X64-M52xlarge-Docker-Builder'
+            image 'opensearchstaging/ci-runner:ubuntu2004-x64-docker-buildx0.6.3-qemu5.0-awscli1.22-jdk11-v2'
+            args '-u root -v /var/run/docker.sock:/var/run/docker.sock'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(
+            name: 'PACKER_AGENT_JSON_NAME',
+            description: 'The JSON file name of type of Jenkins agent this job is building. e.g. jenkins-agent-al2-arm64.json. Files exist in https://github.com/opensearch-project/opensearch-ci/tree/main/packer',
+            trim: true
+        )
+    }
+    stages {
+        stage('Parameters Check') {
+            steps {
+                script {
+                    if (PACKER_AGENT_JSON_NAME.isEmpty()) {
+                        currentBuild.result = 'ABORTED'
+                        error("PACKER_AGENT_JSON_NAME parameter cannot be empty.")
+                    }
+                    sh ("curl -SLO \"https://raw.githubusercontent.com/opensearch-project/opensearch-ci/main/packer/$PACKER_AGENT_JSON_NAME\"")
```

Comment:
I believe adding -f will hard fail this so you dont have to check for validation again